crimsobot/cogs/utilities.py

```diff
@@ -95,6 +103,10 @@ async def polltally(self, ctx: commands.Context, poll_id: Optional[str] = None)
                     poll_found = True
                     descr = message.embeds[0].description
                     reactions = message.reactions
+
+                    if message.embeds[0].image is not discord.Embed.Empty:
+                        imageurl = message.embeds[0].image.url  # this looks kind of yucky
```

Comment:
I don't like concatenating words with variable names and would rather separate them with an underscore per convention; i.e `image_url`, but this is up to choice.